pkg/reconciler/v1alpha1/broker/broker.go

```diff
@@ -217,7 +218,7 @@ func (r *reconciler) reconcile(ctx context.Context, b *v1alpha1.Broker) (reconci
 
 	svc, err := r.reconcileIngressService(ctx, b)
 	if err != nil {
-		logging.FromContext(ctx).Error("Problem reconciling ingress Service", zap.Error(err))
+		logging.FromContext(ctx).Error("Problem reconciling ingress service", zap.Error(err))
```

Comment:
I would probably leave Service capitalized as it represents a K8s Service.

Reply:
Will do!